Formula/bazarr.rb

```diff
@@ -0,0 +1,81 @@
+class Bazarr < Formula
+  include Language::Python::Virtualenv
+
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  url "https://github.com/morpheus65535/bazarr/releases/download/v1.0.3/bazarr.zip"
+  sha256 "540a5c4d80110ba35a566c17829cd6f31444db7ade7c6b9d0603eccaaed8a628"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "zlib"
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/3b/94/e2b1b3bad91d15526c7e38918795883cee18b93f6785ea8ecf13f8ffa01e/lxml-4.8.0.tar.gz"
+    sha256 "f63f62fc60e6228a4ca9abae28228f35e1bd3ce675013d1dfb828688d50c6e23"
+  end
+
+  resource "webrtcvad-wheels" do
+    url "https://files.pythonhosted.org/packages/1c/37/56ddf05b6eaf4023b2d3eb069954fd9150b452ee326d0ea20af1d0d4b0c2/webrtcvad-wheels-2.0.10.post2.tar.gz"
+    sha256 "151bf3998fb731afff90dba77808326235370a6bb467a2d1b81345b10d1de10d"
+  end
+
+  def install
+    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
+    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
+
+    pyver = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{pyver}/site-packages"
```

Comment:
There is helper `site_packages` for this:
```suggestion
    ENV.prepend_create_path "PYTHONPATH", libexec/Language::Python.site_packages("python3")
```

Reply:
Thanks, I didn't know. There should really be an effort to convert a lot of existing formulae to use that, since most of them don't still.